### PR TITLE
Remove PG code from pgduckdb_seq_scan

### PR DESCRIPTION
**Pull request by Y--**

### CODE REVIEW

File: include/pgduckdb/pg/declarations.hpp [hunk 1]
```diff
@@ -30,8 +36,13 @@ typedef HeapTupleData *HeapTuple;
 
 struct Node;
 
+typedef uint16_t OffsetNumber;
+
 typedef unsigned int Oid;
 
+typedef char *Pointer;
```
Comment (JelteF) [inline include/pgduckdb/pg/declarations.hpp:43]: Maybe it makes sense to include `c.h`, which includes a bunch of these definitions.

File: src/scan/postgres_seq_scan.cpp [hunk 2]
```diff
@@ -98,7 +100,7 @@ PostgresSeqScanFunction::PostgresSeqScanFunc(duckdb::ClientContext &context, duc
 
 	auto hasTuple = local_state.m_heap_table_reader->ReadPageTuples(output);
 
-	if (!hasTuple || local_state.m_heap_table_reader->GetCurrentBlockNumber() == InvalidBlockNumber) {
+	if (!hasTuple || IsValidBlockNumber(local_state.m_heap_table_reader->GetCurrentBlockNumber())) {
```
Comment (JelteF) [inline src/scan/postgres_seq_scan.cpp:103]: Probably the reason for the failing tests:
```suggestion
	if (!hasTuple || !IsValidBlockNumber(local_state.m_heap_table_reader->GetCurrentBlockNumber())) {
```

Comment (Y--) [reply src/scan/postgres_seq_scan.cpp:103]: Ah yeah that would help :-) - I'm waiting for #421 to land to take care of this one

File: include/pgduckdb/pg/declarations.hpp (hunk 1 above)
Comment (Y--) [reply include/pgduckdb/pg/declarations.hpp:43]: You think it's preferable rather than only bringing here what we need?

**Review by JelteF — APPROVED** [review]

File: include/pgduckdb/pg/declarations.hpp (hunk 1 above)
Comment (JelteF) [reply include/pgduckdb/pg/declarations.hpp:43]: To be clear, I think it should be completely safe to include. But it does pollute the global namespace with a bunch of things that we don't need.

**Pull request merged**